### ISSUE TITLE
Feature Request: Quick-add cards without navigation

### ISSUE THREAD

**Issue by Tohsig** [NicoSantangelo/sublime-text-trello#7]
First off, plugin works brilliantly. Nicely done :D 

What I would love is to be able to create a card directly from the top-level command palette without having to go through "Navigate." In my mind, the workflow would look something like this: 
- The "Active" list would be the last Board -> List that a user selected through the current `Trello: Navigate` menu option.
- `Trello: Quick create card` and `Trello: Create card with description` would be available from the top-level commands (as well as where they currently are under `Trello: Navigate`). 
- Selecting either of those options would automatically send a card to the currently active Board/List. 

Basically this is similar to what's present in [SublimeGit](https://sublimegit.net/). There I can do the full `Git: Commit` command, or just run the `Git: Quick Commit` option to skip the more complex options.

Haven't written any plugins for ST yet, so I'm not sure how complex that would be to add. Thoughts?
